OWNERS

```diff
@@ -1,11 +1,8 @@
-# See the OWNERS docs: https://git.k8s.io/community/contributors/devel/owners.md
-owners:
-  - droot
-  - pwittrock
```

Comment:
According to https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#owners, the file does nto accept any `owner` section.